polkadot/runtime/parachains/src/configuration.rs

```diff
@@ -1194,6 +1198,21 @@ pub mod pallet {
 				config.minimum_backing_votes = new;
 			})
 		}
+		/// Set availability chunk shuffling params.
+		#[pallet::call_index(53)]
+		#[pallet::weight((
+			T::WeightInfo::set_config_with_u32(), // TODO: add set config with bool
+			DispatchClass::Operational
+		))]
+		pub fn set_availability_chunk_shuffling_client_feature(
```

Comment:
A more future proof solution is to function that takes the `u64` as input and overwrites the configuration, so we could use it to enable/disable this and future client functionality. The concern I have is that it becomes quite a dangerous interface to use, since it is so easy to flip a bit.

Reply:
Yes, this is exactly what I wanted to avoid - I didn't want to expose a bitfield as this is a rather complicated interface to regular users and it's easy to misuse.

I think it's better to have explicit getters and setters for each bit

Reply:
Right, but having it this way we loose all the generecity, so for each bit we have to always comeback here and add a new runtime API.

What about having something like:
`enable_feature(bit_number)`
`disable_feature(bit_number)`

Reply:
I don't think we lose all the genericity. The runtime API is called `client_features` and returns the entire `ClientFeatures` bitflags wrapper.

The only change would be that if a new feature is added, a new extrinsic type needs to be added to the configuration pallet.

> What about having something like:
`enable_feature(bit_number)`
`disable_feature(bit_number)`

Then this would raise questions about endianness

I see your point but I think that the bitflags should be an implementation detail and not part of the API. That's my opinion but I'm open to changing it if majority thinks differently.

I'll add this ClientFeatures thingy to a new PR as @alexggh suggested and we can gather more feedback there